GVFS/GVFS.Common/GVFS.Common.csproj

```diff
@@ -22,8 +23,10 @@
   <ItemGroup>
     <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="1.0.165" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
-    <PackageReference Include="StyleCop.Error.MSBuild" Version="1.0.0" Condition="'$(OS)' == 'Windows_NT'" />
-    <PackageReference Include="StyleCop.MSBuild" Version="5.0.0" Condition="'$(OS)' == 'Windows_NT'" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.0.2">
+      <PrivateAssets>all</PrivateAssets>

```

Comment:
Why are these lines needed only for .NET Core/SDK style projects but not in Framework projects?

Reply:
.NET Core consume NuGet package via the `PackageReference`, and the `PrivateAssets` and `IncludeAssets` control how dependencies are handled (see [here](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets)). .NET Framework projects can use `PackageReference`, but defaults to using packages.config.

Reply:
Reading that page closely, it looks like all we should need is `PrivateAssets = all` since PrivateAssets implies that the items will be consumed like IncludeAssets. 
```
Tag | Description | Default Value
IncludeAssets | These assets will be consumed | all
ExcludeAssets | These assets will not be consumed | none
PrivateAssets | These assets will be consumed but won't flow to the parent project | contentfiles;analyzers;build
```

This page makes it sound like just adding `PrivateAssets="All"` to our PackageReference is good enough. Does that not work for us? https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/DotNetCli.md

Reply:
Your reading seems reasonable. The existing configuration was written out by the NuGet package manager. I can remove the `IncludeAssets` line and verify it still works.

Reply:
Super nitpick: You can do this all in one line, which I think is cleaner:
```
    <PackageReference Include="StyleCop.Analyzers" Version="1.0.2" PrivateAssets="all"> 
```